specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -34,6 +34,54 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "CheckAfdEndpointNameAvailability"
```

Comment:
Lets use name CheckEndpointNameAvailability. This logic can be reused for CDN endpoint as well. Rename the other models accordingly.

Reply:
Renamed the API to CheckEndpointNameAvailability